src/components/CloudHeader/_cloud-header.scss

```diff
@@ -0,0 +1,119 @@
+@import 'carbon-components/scss/globals/scss/colors';
+@import 'carbon-components/scss/globals/scss/helper-mixins';
+
+$cloud-header-link-color: $inverse-01 !default;
+$cloud-header-icon-color: $inverse-01 !default;
+$cloud-header-menu-color: $brand-02 !default;
+$cloud-header-background-color: $color__blue-90 !default;
+
+.cloud-header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: $cloud-header-background-color;
+  color: $cloud-header-link-color;
+  height: rem(48px);
+  width: 100%;
+
+  a {
+    color: $cloud-header-link-color;
+    text-decoration: none;
+    transition: color $transition--base $bx--standard-easing;
+
+    &:hover,
+    &:focus {
+      color: $cloud-header-menu-color;
+    }
+  }
+}
+
+.cloud-header__container,
+.cloud-header-brand {
+  height: 100%;
+  display: flex;
+  align-items: center;
+}
+
+.cloud-header__app-menu {
+  @include button-reset;
+  box-sizing: content-box;
```

Comment:
Just curious, what is this sizing model used for?

Reply:
Just to ensure consistent sizing / padding across browsers

Reply:
Right, I always thought that was `border-box` so was curious how `context-box` differed 😄

Reply:
`border-box` was giving me some odd issues, but this seemed to resolve it >_<